### DISCUSSION TITLE
Does milkdown provide the support for translation

### DISCUSSION THREAD

**Discussion by divya-jeevanantham** [Milkdown/milkdown#1251] [Q&A]
Does Milkdown provide support for translation of the text within built-in components, such as the link preview functionality (paste link, confirm)?

**Comment by zgrybus** [comment]
@DivyaAmar I believe, that translations for Milkdown are not needed at all, since this library just simply maps your markdown input into text output. 
You may translate your input by yourself :P

**Reply by divya-jeevanantham** [reply]
Yes it is required @zgrybus. For example, text used in insert URL popup.

**Reply by zgrybus** [reply]
@DivyaAmar You may change the text by yourself. Milkdown just maps your input 🤔 May you share the code?
